scripts/woltka_merge

```diff
@@ -19,7 +19,9 @@ def merge(prep, base, glob, name, rename):
     prefix_to_name = prep.set_index('run_prefix')['sample_name'].to_dict()
 
     search = os.path.join(base, glob)
-    tables = [biom.load_table(f) for f in glob_.glob(search)]
+    tables = [biom.load_table(f) for f in glob_.glob(search)
+              if len([1 for x in open(f)]) > 1]
```

Comment:
Should the call to open be replaced for exists? `open` is otherwise going to leave a bunch of file handles open, right?

Reply:
Sadly, no ... the files exist but are "empty" (only one line) ...  I could do:
```python
tables = []
for for f in glob_.glob(search): 
    with open(f) as fh:
        len_fh = len([1 for x in fh])
    if len_fh > 1:
        tables.append(biom.load_table(f))
```

Reply:
What about using the file's size: https://stackoverflow.com/a/2507871

Bonus, I think, it would allow to keep everything in a list comprehension loop.

Reply:
My concern is that we don't know what's the minimal size as it will depend on the number of rows and sample names ...

Reply:
I thought just knowing the size was greater to zero was enough, is that not the case? This is a text file, right?

Reply:
Yes a text file in tsv format, the issue is that it fails when the file has less than 2 lines ... AKA only the header is present.

Reply:
Thanks for the explanation. Then your suggestion works well, only issue is that it would read through the whole file, any chance we could break after line 2?